content/eng/central/commerce_and_industries/AIC.md

```diff
@@ -0,0 +1,68 @@
+        b. Cost components mentioned below in 6 shall not be considered as eligible project costs.
+        c. The budged plan presented by the applicant should clearly demarcate all components of the project costs which are not eligible for funding by the AIM, including those mentioned in 6.
+    3. Key aspects of funding AICs proposed by applicants other than academic/ R&D institutes:
+        a. AICs would be provided a grant-in-aid of upto 50% of total project, subject to maximum of Rs. 10 crore.
+        b. Rental value of the building as per fair market value or as per the registered lease agreement can be considered as the applicant's contribution to the AICs in the eligible project cost.
+        c. Costs mentioned in 6 shall not be considered as eligible project costs.
+    4. The grant-in-aid shall be disbursed in a phased manner as per the budget plan submitted by the applicant and approved by AIM directorate.
+    5. Budget Guidelines:
+        a. Expenditure on office furnishings, administration, utility, maintenance and office expenses should be kept to minimum.
+        b. The larger part of the grant-in-aid should be spent on the core operation activities of AICs.
+        c. The total grant-in-aid which can be spent towards sector specific laboratory and seed fund combined should not exceed Rs. 4.5 crores.
+    6. The following elements shall not be allowed to be included in the project cost itself:
+        a. Cost of purchasing land, construction of building, purchase of vehicles.
+        b. Rental expense in case of applicant is an academic institution and the AIC is located in the premises of the academic institution.
+        c. Cost of hiring any external agency/ consultant for developing the AIC operational model or for conducting whole or part of the AIC operations.
+        d. Cost of any fee paid to any external agency/individual for the preparation of the application for AIC.
+
+#### Infrastructure Requirements:
+
+    1. The applicant would have to provide at least 10,000 sq. ft. of ready-to-use, built-up space, for the exclusive use of the AIC. The space should be owned or leased by applicant.
+    2. The space committed should be ready for use and should not require any civil works/ renovations before use.
+    3. The cost of the land and building shall not be included in the project cost. Only in case of non-academic applicants, the rental value of the building as per fair market value or as per the lease agreement shall be considered as the applicant's contribution to the project cost.
+
+These are certain terms and conditions to avail **Atal Incubation Centres** as your own startup funded by Government.
+
+If you want to read full guidelines for **Atal Incubation Centres**. You can have a look at [AIM][1]
+
+[1]: https://s3.ap-south-1.amazonaws.com/thepolicy/policyFiles/AIC_Guidelines_2017.pdf
```

Comment:
provide more reference from vikaspedia, or can be other gov docs on this subject, we need more backlinking to increase reach of this site

Reply:
reference to NITI Aayog official site for Atal Incubation Centres